src/Components/sidebar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styles from './sidebar.module.css'
+import { SidebarData } from './SidebarData'
```

Comment:
rename SidebarData to sideBarData

Reply:
renamed it to just "data" since it will hold the data for the other pages as well.